shoop/simple_pricing/admin_form_part.py

```diff
@@ -69,7 +69,7 @@ def _process_single_save(self, shop, group):
         if clear:
             SimpleProductPrice.objects.filter(product=self.product, group=group, shop=shop).delete()
         else:
-            spp, _ = SimpleProductPrice.objects.get_or_create(product=self.product, group=group, shop=shop)
+            spp, _ = SimpleProductPrice.objects.get_or_create(product=self.product, group=group, shop=shop, price=value)
             spp.price = value
             spp.save()
```

Comment:
Is this really correct? Why the price is assigned twice? Shouldn't the get_or_create just specify the key fields (or if it specifies data fields too, then with the `default` dictionary)? And since the price is set and saved below, this seems redundant anyway.